tests/integration/iop-remote-integration-test-defaults.yaml

```diff
@@ -14,12 +14,6 @@ spec:
       enabled: false
     istiodRemote:
       enabled: true
-    ingressGateways:
```

Comment:
@stevenctl sorry I ignored this part . We can not have the gateway enabled when we install on remote from the beginning. The reason here is the ingress gateway can not be started after the primary cluster patched related namespace.

Reply:
Are you sure that's the case? This code that deploys this runs after all the primary cluster setup happens...

Reply:
https://github.com/istio/istio/blob/master/pkg/test/framework/components/istio/operator.go#L286 . 

we create the secret after we deploy the remote cluster. and then it is time to deploy gateway.